src/autoscaler/metricscollector/exampleconfig/example.yml

```diff
@@ -12,5 +15,4 @@ db:
   metrics_db_url: "postgres://postgres:postgres@localhost/autoscaler?sslmode=disable"
 collector:
   refresh_interval_in_seconds: 60
```

Comment:
can 60s decoded as a Duration directly?  I tested and seems not.  

or you suggested to load as a string and then transform to Duration using ParseDurations?